source/simulator/parameters.cc

```diff
@@ -1262,6 +1262,14 @@ namespace aspect
                          "This method can only be chosen if melt transport is active in the "
                          "model."
                          "\n"
+                         "\\item ``darcy field'': If a compositional field is marked with this "
+                         "method, then its values are computed in each time step by "
+                         "advecting along the values of the previous time step using the "
+                         "fluid velocity prescribed by Darcy's Law, and applying reaction rates "
+                         "to it. In other words this corresponds to the usual notion of a composition "
+                         "field as mentioned in Section~\\ref{sec:compositional}, except that it is "
+                         "advected with the Darcy velocity instead of the solid velocity."
```

Comment:
Please add a line that says "This advection method requires a compositional field with name and type porosity to exist in the simulation, and this field needs to be advected with the "darcy field" method as well."